nsxiv.1

```diff
@@ -433,7 +433,7 @@ and the arguments given to it (where "Optional" arguments might be empty) are:
 2) current file number,
 3) total file count,
 4) image width (Optional: Disabled on thumbnails mode),
-5) image height (Optional: Disabled on thumbnails mode),
+5) image height (Optional: Disabled on thumbnails mode), and
```

Comment:
This part of the man page needs to be synced to the comments on the example script.

The example script lists all these out one per line, so the `and` isn't necessary. I think we should do the same on the manpage instead of adding `and`.